manager/controllers/utils/configuration.go

```diff
@@ -22,9 +22,6 @@ const (
 	VaultUserHomeKey                    string = "VAULT_USER_HOME"
 	VaultDatasetHomeKey                 string = "VAULT_DATASET_HOME"
```

Comment:
Should be removed.

Reply:
I'll remove VaultUserHomeKey. Still needs VaultDatasetHomeKey